src/components/crossForm/SuccessMsg.vue

```diff
@@ -3,15 +3,14 @@
     v-if="show"
     id="success"
     class="mt-3 align-center text-center alert alert-dismissible fade show"
-    style="display:none;"
   >
     <div class="outline-rounded">
       <div style="font-size: 32px;"><i class="fas fa-check"></i></div>
       <div>
         You will receive
-        <span id="receive" class="black"> {{ receiveAmount }} {{ receiveToken }} </span>
-        in your wallet in {{ confirmations.block }} blocks
-        <span id="confirmationTime"> aproximately {{ confirmations.time }}</span>
+        <span id="receive" class="black"> {{ valueAmount }} {{ tokenName }} </span>
```

Comment:
Ah, just noticed: this message is misleading. The user needs to claim for the tokens after that amount of blocks/minutes. I'd rephrase this to something like: "You will be able to claim for your {{valueAmount}} {tokenName}, after {{blocks}} blocks (approximately {{time}} ". 
You can also add a link to the transactions page so the user can navigate directly from this message.